src-ocaml/baijiu_blake2b.ml

```diff
@@ -162,14 +162,12 @@ module Make (B : Baijiu_buffer.S)
        ; param.personal.(12) land 0xFF
        ; param.personal.(13) land 0xFF
        ; param.personal.(14) land 0xFF
-       ; param.personal.(15) land 0xFF |]
-    in
-    let res = B.create 64 in
+       ; param.personal.(15) land 0xFF |] in
 
-    for i = 0 to 63
-    do B.set res i (Char.unsafe_chr (Array.get arr i)) done;
+    let res = Bytes.create 64 in
```

Comment:
I think (personal opionion) `Bytes.create` should be deprecated/renamed to `Bytes.unsafe_create` and replaced with `Bytes.init` wherever possible. How about replacing the lines below with something like:
```ocaml
let res = Bytes.init 64 (fun i -> Char.unsafe_chr (Array.get arr i))
```